### DISCUSSION TITLE
Gradient color in for histogram?

### DISCUSSION THREAD

**Discussion by sushanyadav** [tradingview/lightweight-charts#1222] [Q&A] [answered]
Does Historygram support gradient color?
This is basically what I want to achieve:
<img width="786" alt="CleanShot 2022-12-12 at 3 25 51@2x" src="https://user-images.githubusercontent.com/53527664/207012682-7738962c-4ca6-4187-9de9-f154507af7d6.png">

**Comment by romfrancois** [comment, marked as answer]
Hi @sushanyadav !

Unfortunately no, only rgb(a).

**Reply by sushanyadav** [reply]
Do we have support for rounded corners as seen in the picture above?

**Reply by romfrancois** [reply]
That's not something we support.
